scripts/lua/modules/notifications/endpoints/webhook.lua

```diff
@@ -57,18 +55,64 @@ end
 
 -- ##############################################
 
+-- Format alert into a standardized structure
+local function formatAlert(alert_json)
+  local alert = json.decode(alert_json)
+  if not alert then return nil end
+  
+  -- Create a standardized alert structure
+  return {
+    id = alert.alert_id,
+    timestamp = alert.tstamp,
+    timestamp_end = alert.tstamp_end,
+    type = alert_consts.alertTypeLabel(tonumber(alert.alert_id), true, alert.entity_id),
+    severity = alert.alert_severity,
+    score = alert.score,
+    entity = {
+      type = alert.entity_id,
+      value = alert.entity_val
+    },
+    message = alert.alert,
+    details = alert.info,
+    source = {
+      interface_id = alert.interface_id,
+      probe_ip = alert.probe_ip
+    },
+    metadata = alert.metadata or {}
+  }
+end
```

Comment:
It is not possible to change the format of the notification due to retrocompatibility reasons.